src/components/ExploreCategories/ExploreCategories.js

```diff
@@ -23,7 +27,17 @@ export const ExploreCategories = () => {
             {categories &&
               categories.map((category, id) => {
                 return (
-                  <div className="category-card" key={id}>
+                  <div
+                    className="category-card"
+                    key={id}
+                    onClick={() => {
+                      dispatch({
+                        type: "FILTER_BY_CATEGORIES",
+                        payload: category,
```

Comment:
Maybe you should clear the existing filters first and then dispatch this action.

Reply:
done. thank you